roles/edpm_ovs/tasks/download_cache.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Downloading required packages
+  ansible.builtin.dnf:
+    name: "{{ edpm_ovs_packages }}"
+    download_only: true
```

Comment:
is state:latest not needed here?

Reply:
that was exactly the source problem for ovs packages repos conflicts blocking adoption development

Reply:
yes the latest is imporant

what im trying to fix here is the fact that if we do a dnf update we have no ability to pin the package to a specific version

by swapping to latest "openvswitch" will install the latest version available
but we can override the default packages with openvswtich27 or openvswitch32
to pin to 2.7 or 3.2

my assertion is we should use latest whenever we are installing packages and then pin the package via ansible_vars if required during adoption

this approach means that minor update is just a redeployment.

Reply:
@SeanMooney would you please update to latest?